adev/src/app/core/constants/links.ts

```diff
@@ -12,3 +12,4 @@ export const MEDIUM = 'https://blog.angular.dev';
 export const YOUTUBE = 'https://www.youtube.com/angular';
 export const DISCORD = 'https://discord.gg/angular';
 export const BLUESKY = 'https://bsky.app/profile/angular.dev';
+export const StackOverflow = 'https://stackoverflow.com/questions/tagged/angular';
```

Comment:
If we add it, let's remain consistent. 
```suggestion
export const STACKOVERFLOW = 'https://stackoverflow.com/questions/tagged/angular';
```

Reply:
@JeanMeche  We’re currently exporting each link as individual constants and then re-declaring them as readonly properties.

Do you think it would make sense to group them into a single object and use as const? That way we could avoid repeating properties and keep everything in one place.